src/generate_wrapper.py

```diff
@@ -639,11 +638,8 @@ def process_docstring(f):
         for param in f["parameters"]:
             param_type = adapt_param_type(param["type"])
             # remove const and &
-            param_type = param_type.replace("const", "")
             param_type = param_type.replace("Standard_Boolean &", "bool")
```

Comment:
@tpaviot You're right, I didn't include it in the fix_type function because I didn't know all the side effects, that's why I left it outside the function, notice that fix_type is used in other parts of the code, and you didn't that replace on those parts. I tried not to modify much the other code, just to join the similar code in a function to reduce the redundant code. But sure, no problem I will include it.